HOWTO.md

```diff
@@ -591,3 +635,59 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
     ```
 
 3. Run the client: `fdo-client-linuxappp`
+
+### Environmental Variables
+
+Here is a list of environmental variables, their meaning and suggested values. 
+Note: Contributors/Developers are encouraged to amend to this list of env variables 
+as they come across or write new variables.
+
+1. `LOG_LEVEL` 
+- Usage: This variable is used to provide the log level while running any of the 
+servers or client binary or systemd service.
+- Suggested values:
+`trace` , `info`, `debug`, `error`
+
+2. `MANUFACTURING_SERVER_CONF`
+- Usage: This variable is used to provide path to `manufacturing_server.yml` config file
+- Suggested values: `/path/to/manufacturing_server.yml`
+
+
+3. `DI_MFG_STRING_TYPE` 
+- Usage: This variable is used to provide what type of device info is needed from the device.
+This can be used for device identification.
+As of today this variable has two values mentioned below.
+- Suggested values: `SerialNumber` or `StructuredDeviceInfo`
+
+Note: 
+`SerialNumber` - provides device identification in the form of a string provided by manufacturer
+which is read from `/sys/devices/virtual/dmi/id/product_serial` or `/sys/devices/virtual/dmi/id/chassis_serial`
+locations inside the device.
+`StructuredDeviceInfo` - whereas this option allows user to get device info in the form of a string
+which consists of MAC addresses of the network devices. Depending upon the availability of the
+value of `SerialNumber` from manufacturer this string may also be present in this information.
+
+4. `DI_KEY_STORAGE_TYPE`
+- Usage: This variable is used to specify which type of key storage medium is to be used while
+running `fdo-manufacturing-client` 
+- Suggested values: `FileSystem` or `Tpm`
```

Comment:
This variable has two possible values: `FileSystem` or `Tpm`, and then remove the "Suggested values" line,